runtime/tchannel_server.go

```diff
@@ -74,16 +76,29 @@ func NewTChannelRouter(registrar tchannel.Registrar, logger *zap.Logger) *TChann
 
 // Register registers the given TChannelHandler to be called on an incoming call for its method.
 // "service" is the thrift service name as in the thrift definition.
-func (s *TChannelRouter) Register(service string, method string, h TChannelHandler) {
-	handler := &handler{tchannelHandler: h}
+func (s *TChannelRouter) Register(
+	service, method string,
+	h TChannelHandler,
+	scope tally.Scope,
```

Comment:
The semantics here is messy, with `Register(service string, method string, h TChannelHandler)` it's clear that I register a handler for the given service and method. Scope should made accessible from the `TChannelHandler`, and you could do that by adding a `Scope()` method to the `TChannelHandler` interface.

Reply:
Yes, I agree. Adding `Scope()` method to `TChannelHandler` interface would be better.